### PR TITLE
Add tests for global.go

### PR DESCRIPTION
**Pull request by smoreg**
* Tests
* Small refactoring to make test possible
* Small bug fixes

PR for #2

### CODE REVIEW

**Comment by shilangyu** [comment]
This will conflict with #3, could you wait for it to be merged and rebase from there?

**Comment by shilangyu** [comment]
#3 got merged, could you adjust to it?

**Comment by smoreg** [comment]
@shilangyu review plz. Sorry for some refactoring.

File: cmd/global.go [hunk 1]
```diff
@@ -31,46 +35,75 @@ var globalCmd = &cobra.Command{
...
+		cwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		if !ok {
+			return errRuntimeCaller
+		}
+		absPath := getAbsPathFrom(args[0], cwd)
+		currDir, _ := path.Split(currFile)
+		globalDriveignoreAbsPath := getAbsPathFrom(globalDriveignorePath, currDir)
```
Comment (shilangyu) [inline cmd/global.go]: this shouldnt be here, #3 added the config dir (here `globalDriveignorePath`) and we no longer care about the working directory. Please look how #3 did it.

Comment (shilangyu) [reply cmd/global.go]: This seems to be the reason why tests are failing. I get the following error:
```sh
 Error:          Received unexpected error:
                                open C:\Coding\driveignore\cmd\C:\Users\Lenovo\AppData\Local\Temp\driveignore_Test_globalRun_file_n1_402933465: The filename, directory name, or volume label syntax is incorrect.
```

**Review by shilangyu — CHANGES_REQUESTED** [review]

File: cmd/global.go (hunk 1 above)
Comment (smoreg) [reply cmd/global.go]: @shilangyu my bad,  fix pushed, check pls

**Comment by smoreg** [comment]
@shilangyu

File: utils/path.go [hunk 2]
```diff
@@ -40,3 +40,12 @@ func Walker(path string, walk func(string, os.FileInfo, string) error) error {
...
 	})
 }
+
+// GlobalDriveignorePath returns absolute path to .global_driveignore
+func GlobalDriveignorePath() string {
+	dir, err := os.UserConfigDir()
+	if err != nil {
+		panic(err)
+	}
+	return filepath.Join(dir, ".global_driveignore")
```
Comment (shilangyu) [inline utils/path.go]: Please change to `UserConfigDir/driveignore/.global_driveignore`

**Review by shilangyu — CHANGES_REQUESTED** [review]
Last change and its all good! Thanks for the help!

**Comment by shilangyu** [comment]
@smoreg

File: utils/path.go (hunk 2 above)
Comment (smoreg) [inline utils/path.go]:
```suggestion
	return filepath.Join(dir, "/driveignore/.global_driveignore")
```
Something like this?

Comment (smoreg) [reply utils/path.go]: @shilangyu update with suggestion

**Review by shilangyu — APPROVED** [review]
awesome, thanks!

**Pull request merged**